crates/sui-adapter-transactional-tests/tests/random/move_call_clock_after_random_and_transfer.move

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --accounts A B --addresses test=0x0
+
+//# publish --sender A
+module test::random {
+    use sui::clock::Clock;
+    use sui::random::Random;
+
+    public fun use_clock(_clock: &Clock) {}
+    public fun use_random(_random: &Random) {}
+}
+
+// bad tx - use_random, transfer, use_clock
+//# programmable --sender A --inputs 10 immshared(8) immshared(6) @B
```

Comment:
can you use random as an input more than once? Might be a good test?
```
//# programmable --sender A --inputs 10 immshared(8) immshared(8)
//> test::random::use_random(Input(0));
//> test::random::use_random(Input(1));
```

Reply:
done (though with `Input(0)` twice)